sktime/forecasting/base/adapters/_pytorchforecasting.py

```diff
@@ -39,7 +39,11 @@ class _PytorchForecastingAdapter(_BaseGlobalForecaster):
         parameters to be passed for `TimeSeriesDataSet.to_dataloader()`
         by default {"train": False}
     model_path: string (default=None)
-        try to load a existing model without fitting.
+        try to load a existing model without fitting. Calling the fit function is
```

Comment:
I wonder, if we should also allow to pass an initialized pytorch-forecasting model and not only a path where it is stored. 

Any opinions? Could also be solved via a different PR.

Reply:
> I wonder, if we should also allow to pass an initialized pytorch-forecasting model and not only a path where it is stored.

Yeah, it could be done by passing a model and assigning it to `self.best_model`.

Reply:
We can make it capable of fine tuning actully, loading from a existing model and than tuning it. I am not sure whether it's consistant with the fine tuning api design though, maybe we can wait until the fine tuning dessign is finished.

Reply:
I am not sure, if these models can be easily used within fine-tuning, since for fine-tuning we need direct access to the neural network so I am unsure if this is possible with pytorch-forecasting easily.

I think it would be cool, if you can create an issue for adding support to pass a model directly.

Reply:
issue is created: https://github.com/sktime/sktime/issues/6736